.eslintrc

```diff
@@ -14,3 +14,4 @@ rules:
   no-use-before-define: [warn, { functions: false }]
   react/no-multi-comp: [off]
   no-throw-literal: [off]
+  import/no-unresolved: [warn]
```

Comment:
I find this rule really useful, so I think it would be good to not lower it. I'll poke at this and see if I can get it working.